### PR TITLE
etcd-runner: Added connection timeout flag for client

### PR DESCRIPTION
**Pull request by sinsharat**
#6722

### CODE REVIEW

**Comment by sinsharat** [comment]
@xiang90 i have run the test for 10000 rounds with client-timeout=300 seconds running on  flat-injection-cluster. Also i have put logs for ensuring that the test case for watch is executing properly. Its been 1.5hrs since its been running and so far tests are working fine.

**Comment by xiang90** [comment]
lgtm

File: tools/functional-tester/etcd-runner/main.go [hunk 1]
```diff
@@ -43,6 +45,7 @@ func main() {
 	endpointStr := flag.String("endpoints", "localhost:2379", "endpoints of etcd cluster")
 	mode := flag.String("mode", "watcher", "test mode (election, lock-racer, lease-renewer, watcher)")
 	round := flag.Int("rounds", 100, "number of rounds to run")
+	flag.Int64Var(&clientTimeout, "client-timeout", 5, "max timeout seconds for a client to get connection")
```
Comment (sinsharat) [inline tools/functional-tester/etcd-runner/main.go]: @xiang90 sure. makes sense to keep it as 60. Will modify it.

Comment (sinsharat) [inline tools/functional-tester/etcd-runner/main.go]: done.
Thank you!

File: tools/functional-tester/etcd-runner/main.go [hunk 2]
```diff
@@ -43,6 +45,7 @@ func main() {
 	endpointStr := flag.String("endpoints", "localhost:2379", "endpoints of etcd cluster")
```
Comment (xiang90) [inline tools/functional-tester/etcd-runner/main.go]: can you change all other flags to use global vars in another pull requests? so it is consistent with your current change. thanks!

Comment (sinsharat) [inline tools/functional-tester/etcd-runner/main.go]: @xiang90 sure will do it.
Thank you!

Comment (sinsharat) [inline tools/functional-tester/etcd-runner/main.go]: @xiang90 One doubt for rounds and eps do you want them to be passed as variables via functions or to directly use them in the functions since they will be global vars?

Comment (xiang90) [inline tools/functional-tester/etcd-runner/main.go]: yes. just use them directly.

Comment (sinsharat) [inline tools/functional-tester/etcd-runner/main.go]: @xiang90 sure will do it and raise it in another PR as you suggested.
Thank you!

Comment (sinsharat) [inline tools/functional-tester/etcd-runner/main.go]: @xiang90 but for raising them as another PR this needs to be merged first. Or shall i add them here as another commit with different commit title?
